test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/delegatingclients/DelegatingAsyncClientTest.java

```diff
@@ -45,6 +47,8 @@ public class DelegatingAsyncClientTest {
 
     MockAsyncHttpClient mockAsyncHttpClient = new MockAsyncHttpClient();
     ProtocolRestJsonAsyncClient defaultClient = ProtocolRestJsonAsyncClient.builder()
+                                                                           .credentialsProvider(AnonymousCredentialsProvider.create())
+                                                                           .region(Region.US_WEST_2)
```

Comment:
Fixed tests when credentials and region aren't configured. Unrelated to this change.